ml/rlvr/fine_tuning_orchestrator.py

```
@@ -0,0 +1,463 @@
+            "validating_files": 10,
+            "queued": 20,
+            "running": 50,
+            "succeeded": 100,
+            "failed": 0,
+            "cancelled": 0
+        }
+        return status_to_progress.get(status, 0)
+
+    async def _optimize_anthropic_prompt(
+        self,
+        run_id: uuid.UUID,
```

Ruff (0.14.5)

Unused method argument: `run_id`

(ARG002)